packages/grid/_modules_/grid/components/toolbar/ExportSelector.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import MenuList from '@material-ui/core/MenuList';
+import Button from '@material-ui/core/Button';
+import MenuItem from '@material-ui/core/MenuItem';
+import { ApiContext } from '../api-context';
+import { useGridSelector } from '../../hooks/features/core/useGridSelector';
+import { optionsSelector } from '../../hooks/utils/optionsSelector';
+import { GridMenu } from '../menu/GridMenu';
+import { ExportOption } from '../../models';
+
+export function ExportSelector() {
+  const apiRef = React.useContext(ApiContext);
+  const options = useGridSelector(apiRef, optionsSelector);
+  const [anchorEl, setAnchorEl] = React.useState(null);
+
+  const ExportIcon = apiRef!.current.components!.ExportIcon!;
+
+  const ExportOptions: Array<ExportOption> = [
+    {
+      label: 'CSV',
+      format: 'csv',
+    },
+  ];
+
+  const handleExportSelectorOpen = (event) => setAnchorEl(event.currentTarget);
+  const handleExportSelectorClose = () => setAnchorEl(null);
+  const handleExport = (format) => {
+    if (format === 'csv') {
+      apiRef!.current.exportDataAsCsv();
+    }
+
+    setAnchorEl(null);
+  };
+
+  const handleListKeyDown = (event: React.KeyboardEvent) => {
+    if (event.key === 'Tab' || event.key === 'Escape') {
+      event.preventDefault();
+      handleExportSelectorClose();
+    }
+  };
+
+  // Disable the button if the corresponding is disabled
+  if (options.disableCsvExport) {
+    return null;
+  }
+
+  const renderExportOptions: Array<React.ReactElement> = ExportOptions.map((option, index) => (
+    <MenuItem key={index} onClick={() => handleExport(option.format)}>
+      {option.label}
+    </MenuItem>
+  ));
+
+  return (
+    <React.Fragment>
+      <Button
+        color="primary"
+        size="small"
+        startIcon={<ExportIcon />}
+        onClick={handleExportSelectorOpen}
+        aria-label=""
+        aria-haspopup="true"
+      >
+        {'EXPORT'}
```

Comment:
localization...
I guess you know already...

Reply:
yup

Reply:
I thought you were going with one button per export type, but a menu sounds fare too. I was wondering about an icon button too 🤷‍♂️

Reply:
I thought about it. We can have it as a menu for now and see how it will go. Keeping the exports per format separate in hooks will still allow us to switch them on and of based on what is paid and what not.

I added one icon for download that I took from material ui. I was thinking of adding an icon for CSV specifically 🤔 next to the CSV option.

Reply:
Ok for the menu. In this case, could we have it accessible for once :) https://github.com/mui-org/material-ui-x/issues/881

Reply:
I have run a quick research on what online tools are using for the UI of the export to CSV https://github.com/mui-org/material-ui-x/issues/197#issue-683656593. It seems the most of the time `<IconButton arial-label="export">` + `Menu`.

Reply:
The proposed icon looks great. We might need to add a similar URL capability for https://material-ui.com/components/material-icons/ cc @mbrookes

Reply:
Sorry, what's the question?

Reply:
More than a question, an idea: deep linking possible between co-workers, e.g. https://material-ui.com/components/material-icons/?icon=AccessAlarm

Reply:
#881 Is now part of this PR 😄